backend/tournesol/views/suggestions/to_compare.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+from drf_spectacular.utils import OpenApiParameter, extend_schema, extend_schema_view
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+
+from tournesol.lib.suggestions.strategies import ClassicEntitySuggestionStrategy
+from tournesol.serializers.suggestion import EntityToCompare
+from tournesol.views import PollScopedViewMixin
+
+
+class ToCompareStrategy(models.TextChoices):
+    CLASSIC = "Classic"
```

Comment:
I would find more usual to use lowercase in URL parameters
```suggestion
    CLASSIC = "classic"
```